graphene_acl/routing.py

```diff
@@ -16,18 +17,22 @@ def __init__(self, classifier=None):
 
     def register(self, handler, key=None):
         if key:
-            self._handlers[key] = handler
+            if isinstance(key, Enum):
```

Comment:
Do we also need to handle `graphene.Enum` or is that type captured by stdlib `Enum`?

Reply:
ya its captured by Enum. graphene.Enum actually doesn't even work....